ansible/roles/backups/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Install s3cmd
+  ansible.builtin.apt:
+    pkg:
+      - s3cmd
+    update_cache: true
+  become: true
+
+- name: Move s3 config file # needs to be decrypted before its moved
+  ansible.builtin.copy:
+    src: .s3cfg
+    dest: /home/ubuntu/
+    mode: "0755"
+  become: true
+
+- name: Setup primary and standby
+  when: inventory_hostname != "lard-staging"
+  block:
+    - name: Move backup shell scripts
+      ansible.builtin.copy:
+        src: cron_scripts/
+        dest: /home/ubuntu/cron_scripts
+        mode: "0755"
+      become: true
+
+    # backup to s3 in lard/backups bucket
+    - name: Create cron to backup lard DB
+      ansible.builtin.cron:
+        name: "backup lard"
+        minute: "40"
+        hour: "1" # backup weekly at 1:40am (UTC)
+        weekday: "1" # every Monday
+        job: "/home/ubuntu/cron_scripts/lard_backup.sh"
+
+    - name: Create cron to backup lard_restricted DB
+      ansible.builtin.cron:
+        name: "backup lard restricted"
+        minute: "40"
+        hour: "0" # backup weekly at 0:40am (UTC)
+        weekday: "1" # every Monday
+        job: "/home/ubuntu/cron_scripts/lard_restricted_backup.sh"
+
+    - name: Create cron to do base backup
+      ansible.builtin.cron:
+        name: "basebackup lard"
+        minute: "0"
+        hour: "5" # backup monthly at 5:00am (UTC)
+        day: "1" # 1st of every month
+        job: "/home/ubuntu/cron_scripts/base_backup.sh"
```

Comment:
Do we want this active? Given we don't have a workflow for using these?

Reply:
Removed